book_rental.rb

```diff
@@ -0,0 +1,22 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Following the project requirement you are to create a method for `adding rentals`. Which shows that a book can have many rentals, see the piece of code below for guidance :+1: 
- code snippet
```ruby
def add_rental(person, date)
    #the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
end
```